src/request/remove-key/index.html

```diff
@@ -111,4 +140,5 @@ <h1 data-i18n="export-file-heading" class="nq-h1">Download Key File</h1>
 
     </div>
 </body>
-</html>
+
+</html>
```

Comment:
I find it fascinating how your editor just can't seem to keep a trailing line-break... :question: :confused:

Reply:
This still.